ballerina/client.bal

```diff
@@ -1,15 +1,799 @@
-// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
-//
-// WSO2 LLC. licenses this file to you under the Apache License,
-// Version 2.0 (the "License"); you may not use this file except
-// in compliance with the License.
-// You may obtain a copy of the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing,
-// software distributed under the License is distributed on an
-// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-// KIND, either express or implied.  See the License for the
-// specific language governing permissions and limitations
-// under the License.
+// AUTO-GENERATED FILE. DO NOT MODIFY.
+// This file is auto-generated by the Ballerina OpenAPI tool.
```

Comment:
@ovindu-a Shall we use --license option along with the bal openapi command to update the license headers, you can find the license header here https://github.com/ballerina-platform/module-ballerinax-tableau/blob/main/docs/license.txt

You can update this via the below command,

```bash
bal openapi -i docs/spec/openapi.json --mode client  --license docs/license.txt -o ballerina
```

Reply:
Fixed in [6743a6c](https://github.com/ballerina-platform/module-ballerinax-tableau/pull/2/commits/6743a6cca480d2aa332cf707411543d31930409e)